src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -66,6 +66,7 @@ def load_arguments(self, _):
         c.argument('min_count', type=int, validator=validate_nodes_count)
         c.argument('max_count', type=int, validator=validate_nodes_count)
         c.argument('enable_vmss', action='store_true')
+        c.argument('node_zones')
```

Comment:
This should probably leverage the global alias for zones.
```
from azure.cli.core.commands.parameters import zones_type
c.argument('node_zones', zones_type)
```

If you still want to customize on top that I would suggest:
```
c.argument('node_zones', zones_type, options_list='--node-zones', help='Space-separated list of availability zones where agent nodes will be placed.')
```